libs/IMUKit/include/IMUKit.h

```diff
@@ -0,0 +1,53 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <chrono>
+
+#include "CoreAccelerometer.h"
+#include "CoreGyroscope.h"
+#include "Mahony.h"
+#include "interface/libs/EventLoop.h"
+
+namespace leka {
+
+class IMUKit
+{
+  public:
+	IMUKit(interface::EventLoop &event_loop, CoreAccelerometer &core_accelerometer, CoreGyroscope &core_gyroscope)
+		: _event_loop(event_loop),
+		  _core_accelerometer(core_accelerometer),
+		  _core_gyroscope(core_gyroscope) {
+
+		  };
+
+	struct Config {
+		const std::chrono::milliseconds delay {};
+		const float frequency {};
+	};
```

Comment:
This struct `Config` is incomplete

- It is only used to instantiate a private member, therefore it does not have to be public
- If it means to be public, it should at least appears in parameter of one method (`ctor`, `init`, `start`)

Reply:
I missed this review, my bad